xml/obs_best_practice_staging_workflow.xml

```diff
@@ -48,29 +48,29 @@
     different packages all the time. They have to be reviewed, adjusted and
     tested (built) before being accepted. As you can imagine, it becomes
     nonviable to review the packages one by one. Even if the maintainer checks
-    that a package is not broken and merge it, it can break everything else for
+    that a package is not broken and merges it, it can break everything else for
     conflicts with other packages. To deal with these situations, Build Service
     provides what we call Staging Workflow.
   </para>
   <para>
     The idea behind the Staging Workflow is testing the requests incrementally
     by batches. First, a copy of the original project is created, it is called
     Staging Project and is going to act as a playground. The Staging Manager
-    selects some of the requests he consider they belong together and assign the
-    corresponding packages to the Staging Project. This way, the group of
+    selects some of the requests they considers they belong together and assigns the
+    corresponding packages to the Staging Project. This way, the groups of
     packages are going to be tested (built) in one go. Once the Staging Project
     gets built, the changes can be merged to the original project.
   </para>
   <para>
-    The Staging Manager can create as many Staging Projects as he consider and
-    assign different selections of requests to each of them. It is still tedious
+    The Staging Manager can create as many Staging Projects as they considers and
```

Comment:
Grammar fix (they + 3rd person plural verb) and use a better verb than consider:
```suggestion
    The Staging Manager can create as many Staging Projects as they require and
```